common.yml

```diff
@@ -168,7 +168,7 @@ functions:
         if [ "${USE_TLS}" = "true" ]; then
           MONGOD_ARGS="${mongod_args_tls}"
         elif [ "${AWS_AUTH}" = "true" ]; then
-          MONGOD_ARGS="--auth --setParameter authenticationMechanisms=SCRAM-SHA-256"
+          MONGOD_ARGS="--auth --setParameter authenticationMechanisms=MONGODB-AWS,SCRAM-SHA-256"
```

Comment:
@tfogo Seems `MONGODB-AWS` is a valid value according to here: https://docs.mongodb.com/drivers/node/fundamentals/authentication/mechanisms/#mongodb-aws

Reply:
Ah okay! It's strange, it looks like it isn't valid on some distros. Here's MacOS 10.14 throwing an error because of it:

https://evergreen.mongodb.com/task_log_raw/mongo_tools_macos_aws_auth_4.4_patch_57f7f7eafdd68fd8d570dfa1848f7d1312724ed1_60e8dd620ae6067ce0cd2b06_21_07_09_23_36_03/0?type=T#L162